checkov/common/bridgecrew/platform_integration.py

```diff
@@ -77,6 +80,24 @@ def __init__(self):
         self.http = None
         self.excluded_paths = []
 
+    @staticmethod
+    def is_bc_token(token: str) -> bool:
+        return re.match(UUID_V4_PATTERN, token) is not None
+
+    @cached(TTLCache(maxsize=1, ttl=540))
+    def get_auth_token(self) -> str:
+        if self.is_bc_token(self.bc_api_key):
+            return self.bc_api_key
+        # This is a Prisma Cloud token
+        if not self.prisma_url:
+            raise ValueError("Got a prisma token, but the env variable PRISMA_API_URL is not set")
+        [username, password] = self.bc_api_key.split['::']
```

Comment:
```suggestion
        username, password = self.bc_api_key.split('::')
```
I think the square brackets should be be normal round one 😉